quinn-proto/src/connection/streams.rs

```diff
@@ -19,7 +19,8 @@ use crate::{
     Dir, Side, StreamId, TransportError, VarInt, MAX_STREAM_COUNT,
 };
 
-pub(crate) struct Streams {
+#[doc(hidden)]
```

Comment:
Is this attribute still needed, given that the reexport is guarded?

Reply:
nope you're quite right; can't make it _extra_ hidden can we? Removed.